pkg/bootconfig/bootconfig.go

```diff
@@ -87,12 +88,31 @@ func (bc *BootConfig) Boot() error {
 			return err
 		}
 	} else if bc.Multiboot != "" {
+		mbkernel, err := os.Open(bc.Multiboot)
+		if err != nil {
+			log.Printf("Error opening multiboot kernel file: %v", err)
+			return err
+		}
+		defer mbkernel.Close()
+
 		// check multiboot header
-		if err := multiboot.Probe(bc.Multiboot); err != nil {
+		if err := multiboot.Probe(mbkernel); err != nil {
 			log.Printf("Error parsing multiboot header: %v", err)
 			return err
 		}
-		if err := multiboot.Load(true, bc.Multiboot, bc.MultibootArgs, bc.Modules, nil); err != nil {
+		modules := make([]multiboot.Module, len(bc.Modules))
+		for i, cmd := range bc.Modules {
```

Comment:
i feel like this whole opening files act is in a few places... should pkg/boot/multiboot have some utilities for dealing with it?

Reply:
yeah propably, I was thinking 'diskboot' ... but mutliboot probably makes sense, like 
```
func openModules(cmds []string) ([]Modules, error)
```

Reply:
or even a `LoadFromFiles` with the previous prototype of `Load`

Reply:
I think the former would be nice, since it's usable with boot.MultibootImage

Reply:
done